openslides_backend/action/generics/update.py

```diff
@@ -25,9 +25,13 @@ def create_events(self, instance: Dict[str, Any]) -> Iterable[Event]:
         Creates events for one instance of the current model.
         """
         fqid = fqid_from_collection_and_id(self.model.collection, instance["id"])
-        fields = {
-            k: v for k, v in instance.items() if k != "id" and not k.startswith("meta_")
-        }
+        fields = UpdateAction.create_events_for_reuse(instance)
         if not fields:
             return []
         yield self.build_event(EventType.Update, fqid, fields)
+
+    @staticmethod
+    def create_events_for_reuse(instance: Dict[str, Any]) -> Dict[str, Any]:
+        return {
+            k: v for k, v in instance.items() if k != "id" and not k.startswith("meta_")
+        }
```

Comment:
Why not just use a global method defined somewhere accessible from both locations? That would be more pythonic and in line with the rest of the backend.

Reply:
This is global and usable from everywhere, but it is not "somewhere", it is on the place, where it is used in general.

Reply:
Mmh, I'm not a fan. But if you insist, okay. But please use a more appropriate name like `create_update_events`.